### PR TITLE
Subscription e2e tests

### PR DESCRIPTION
**Pull request by selenai**
### Issue number

FRON-310

### CODE REVIEW

File: e2e/helpers/helpers.ts [hunk 1]
```diff
@@ -153,4 +165,51 @@ export class HelpFunctions {
...
   }
+
+  async subsribeToContact(userAuthId: string) {
+    await this.subscriptionInput.click({ clickCount: 3 });
+    await this.page.keyboard.press('Backspace');
+    await this.subscriptionInput.type(`sip:${userAuthId}@voipgrid.nl`);
+    await this.subscribeButton.click();
+  }
+
+  // async assertSubscribedContactStatus(userAuthId: string, contactStatus: ContactStatus) {
```
Comment (Dagg-h) [inline e2e/helpers/helpers.ts:176]: Can be removed?

File: e2e/helpers/helpers.ts [hunk 2]
```diff
@@ -153,4 +165,51 @@ export class HelpFunctions {
     await this.sessionTransferInput.type(number);
     await this.sessionCompleteTransferButton.click();
   }
+
+  async subsribeToContact(userAuthId: string) {
+    await this.subscriptionInput.click({ clickCount: 3 });
+    await this.page.keyboard.press('Backspace');
+    await this.subscriptionInput.type(`sip:${userAuthId}@voipgrid.nl`);
```
Comment (Dagg-h) [inline e2e/helpers/helpers.ts:172]: Here we can use the REALM from the .env.
```suggestion
    await this.subscriptionInput.type(`sip:${userAuthId}@${process.env.REALM}`);
```
Nicer, if we ever test against another realm :)

File: e2e/helpers/helpers.ts [hunk 3]
```diff
@@ -153,4 +165,51 @@ export class HelpFunctions {
...
+  async assertSubscribedContactUri(userAuthId: string) {
+    await expect(this.subscribedContact.nth(0)).toHaveText(`sip:${userAuthId}@voipgrid.nl`);
+  }
+
+  async assertSecondSubscribedContactUri(userAuthId: string) {
+    await expect(this.subscribedContact.nth(1)).toHaveCount(1);
+    await expect(this.subscribedContact.nth(1)).toHaveText(`sip:${userAuthId}@voipgrid.nl`);
+  }
+
+  async assertSubscribedContactStatus(contactStatus: ContactStatus) {
```
Comment (Dagg-h) [inline e2e/helpers/helpers.ts:190]: Maybe nice to make this one more generic, by adding which nth contact you want to check.
For when you want to check the status of your 3rd contact e.g.

File: e2e/helpers/helpers.ts [hunk 4]
```diff
@@ -153,4 +165,51 @@ export class HelpFunctions {
...
+    await expect(this.contactStatus.nth(0)).toHaveText(contactStatus);
+  }
+
+  async assertSecondSubscribedContactStatus(contactStatus: ContactStatus) {
```
Comment (Dagg-h) [inline e2e/helpers/helpers.ts:194]: With above comment, you only need one function!

File: e2e/helpers/helpers.ts [hunk 5]
```diff
@@ -153,4 +165,51 @@ export class HelpFunctions {
...
+  }
+
+  async unsubsribeFromContact(userAuthId: string) {
+    await this.subscriptionInput.click({ clickCount: 3 });
+    await this.page.keyboard.press('Backspace');
+    await this.subscriptionInput.type(`sip:${userAuthId}@voipgrid.nl`);
+    await this.unsubscribeButton.click();
+  }
+
+  async assertContactUnsubscribed() {
```
[the remark this answers is not in the document]
Comment (Dagg-h) [reply e2e/helpers/helpers.ts:206]: Maybe even nicer if we can find the correct element (or not find when it shouldn't be there :p) based on account Id

**Review by patrickswijgman — APPROVED** [review]

**Pull request merged**